### DISCUSSION TITLE
Implementing tiling or other low-VRAM fixes for low-end GPUs

### DISCUSSION THREAD

**Discussion by harracuda** [jhogsett/EMA-VFI-WebUI#241] [Ideas]
For users such as myself on lower-end cards (3060 mobile 6GB in my example), it would be fantastic to see some sort of tiling or similar feature in order to be able to interpolate 1080p and above on these cards that lack signifcant VRAM.

In some cases even 720p source frames will max out VRAM; limiting me to around 540p in actual usage. I suppose tiling with an adjustable tile count (or resolution), then overlap and blending when inference is complete would be a useful implementation.

**Comment by jhogsett** [comment]
@harracuda Thanks for the suggestion. I have also had the idea recently  to use tiling to get around low VRAM. In my case I have a RTX 2080 Ti machine that has trouble with 1080p content, and my RTX 3090 machine has the same problem with 4K content.

Successful interpolation of a _between frame_  between two frames relies on recognizing the same objects in both frames and estimating their motion. With the current full frame interpolation, objects that move too close to the frame edge become unrecognizable, and the interpolation engine will instead pop them into or out of existence in the produced frame. This creates an undesirable side effect. 

I expect the edge artifact problem to be greatly increased if using tiling to interpolate frames in portions. I have abandoned pursuing the idea.
